prep-html-syntax/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Oops! You are missing your `head` element. It's a very understandable thing to miss since there's no content for it in this exercise, but it is still a mandatory part of any HTML document 😁

Reply:
I don't know what to do here. I did push this but now it is saying there is a branch ahead of the origin/master. Please advise. Thank you.